src/examples/omega_copy.js

```diff
@@ -1,5 +1,5 @@
 /**********************************************************************************************************************
- * Copyright (c) 2021-2022 Concurrent Technologies Corporation.                                                       *
+ * Copyright (c) 2021-2022-2022 Concurrent Technologies Corporation.                                                       *
```

Comment:
The date change script got a little carried away on some of these license blurbs.